src/interpreter/executors/main.rs

```diff
@@ -211,9 +211,17 @@ where
             self.tx.set_receipts_root(receipts_root);
         }
 
-        // refund remaining global gas
-        let gas_refund = self.registers[REG_GGAS] * self.tx.gas_price();
+        // used bytes are non-refundable
```

Comment:
?
```suggestion
        // used gas is non-refundable
```

Reply:
I think this comment is explaining why there's no corresponding calculation for bytes when processing the refund?

Reply:
Hmmm. Well if that's the case then it should be worded clearer, that's for sure 😂

Reply:
i'll pick different words, but its not really the gas there. its the price per byte * |witness| that is charged from the base asset, that is non-refundable

Reply:
https://github.com/FuelLabs/fuel-vm/pull/148/commits/3f2bf04821fdda979d48cc15c4f32d7b2dd55057